src/components/atoms/Select.tsx

```diff
@@ -0,0 +1,114 @@
+import DownArrow from "@/page/timetable/assets/DownArrow";
+import color from "@/styles/color";
+import { font } from "@/styles/font";
+import React from "react";
+import styled, { css } from "styled-components";
+
+interface ISelectProps {
+  options: string[];
+  defaultOption: string;
+  label: string;
+  size?: string;
+  handler: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const Select = ({
+  options,
+  defaultOption,
+  label,
+  size = "80px",
```

Comment:
이 size라는 프롭스는 너비 값에만 관여하니깐 width라고 짖는건 어떨까요?
```suggestion
  width = "80px",
```

Reply:
좋습니당